File: Exercícios - Capítulos/Capítulo 8/Atividade 2.py
while True:
    print("\nMenu:\n1 - Quadrado\n2 - Fatorial\n3 - Sair")
    opcao = int(input("Escolha: "))

    if opcao == 1:
        n = int(input("Digite um numero: "))
        print("Quadrado:", n * n)
    elif opcao == 2:
        n = int(input("Digite um numero: "))
        f = 1
        for i in range(1, n+1):
            f *= i
        print("Fatorial:", f)
    elif opcao == 3:
        print("Saindo...")
        break
    else:
        print("Opcao invalida!")

File: Exercícios - Capítulos/Capítulo 8/Atividade 2.c
#include <stdio.h>

int main() {
    int opcao, n;

    do {
        printf("\nMenu:\n1 - Quadrado\n2 - Fatorial\n3 - Sair\nEscolha: ");
        scanf("%d", &opcao);

        switch (opcao) {
            case 1:
                printf("Digite um numero: ");
                scanf("%d", &n);
                printf("Quadrado: %d\n", n * n);
                break;
            case 2:
                printf("Digite um numero: ");
                scanf("%d", &n);
                int f = 1;
                for(int i = 1; i <= n; i++)
                    f *= i;
                printf("Fatorial: %d\n", f);
                break;
            case 3:
                printf("Saindo...\n");
                break;
            default:
                printf("Opcao invalida!\n");
        }
    } while(opcao != 3);

    return 0;
}

